File: C/String/program9.c
//Accept the string from user and display how many times the character contains in the string

#include<stdio.h>

int Display(char str[], char ch)
{
    int iCnt = 0;

    while(*str != '\0')
    {
        if(*str == ch)
        {
            iCnt++;
        }
        str++;
    }
    return iCnt;
}

int main()
{
    char Arr[20];
    char cValue = '\0';
    int iRet = 0;

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    printf("Enter the character that you want to search:\n");
    scanf(" %c",&cValue);

    iRet = Display(Arr,cValue);
    printf("String contains %c : %d times",cValue,iRet);

    return 0;
}

File: Java/String/program9.java
//Accept the string from user and display how many times the character contains in the string

import java.util.*;

class Stringx
{
    public String str;
    public char ch;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();

        System.out.println("Enter the character that you want to search");
        ch = sobj.next().charAt(0);
    }

    public int Count()
    {
        char Arr[] = str.toCharArray();

        int i = 0;
        int iCnt = 0;

        for(i = 0; i < Arr.length; i++)
        {
            if(Arr[i] == ch)
            {
                iCnt++;
            }
        }
        return iCnt;
    }
}

class program9
{
    public static void main(String arg[])
    {
        int iRet = 0;

        Stringx obj = new Stringx();

        obj.Accept();

        iRet = obj.Count();
        System.out.println("String contains the character: "+iRet + " times");
    }
}